containers/ecr-viewer/e2e/dual/admin-user.spec.ts

```diff
@@ -32,4 +34,70 @@ test.describe("user management page", () => {
       );
     expect(nonColorViolations).toEqual([]);
   });
+
+  test("should create a new user", async ({ page, browserName }) => {
+    // Create programs
+    const program1 = await createRandomProgramArea(page);
+    const program2 = await createRandomProgramArea(page);
+
+    // Create user & assign to Program 1
+    await page.goto("/ecr-viewer/admin/user");
+
+    await expect(page.getByText("User management")).toBeVisible();
+
+    await page.getByText("Add new user").click();
+
+    await page.waitForURL("/ecr-viewer/admin/user/create");
+    await expect(page.getByRole("heading", { name: "Add user" })).toBeVisible();
+
+    const accessibilityScanResultsBase = await new AxeBuilder({
+      page,
+    }).analyze();
+    expect(accessibilityScanResultsBase.violations).toEqual([]);
+
+    page.getByLabel("EMAIL").fill(`${browserName}@test.test`);
```

Comment:
with retries is this a safe enough randomization? or should we add a random number on the end or similar?